<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1281</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_r232">
     <attribute name="title">
      <string>Connect</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_rs232">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>200</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_shimming">
     <attribute name="title">
      <string>Shimming</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1241</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_record">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>340</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>RECORD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_frecuency">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>350</y>
        <width>59</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>HZ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_get_shim">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>370</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>GET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_view_signal">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>561</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_frecuency">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ISOCENTRO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>-2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>480</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>UNDO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_config">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_name_equipament">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>GE Profile 4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_number_step">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_filename">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>gemech_001_v.plot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_temp_magnet">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>32.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_create_file">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>300</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Create File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_equipament">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name Equipament</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_number_step">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number Step</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_filename">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_temp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperature Magnet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_file_exit">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>300</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Close File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_temp_room">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>22</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_temp_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperature Room</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
